mod-source-record-storage-client/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.folio</groupId>
     <artifactId>mod-source-record-storage</artifactId>
-    <version>3.2.0-SNAPSHOT</version>
+    <version>3.2.1-SNAPSHOT</version>
```

Comment:
what is the reason for bumping the development version? new endpoint is a backward compatible change, it is definitely not a bugfix, and I strongly oppose incrementing snapshot versions in any case (unless it is absolutely necessary, which is not the case here)

Reply:
@KaterynaSenchenko Sure. Thank you for you explanation!